lib/cli/commands/create.ts

```diff
@@ -0,0 +1,20 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudInitClient } from '../command';
+import { CreateNodeRequest } from '../../proto/xudrpc_pb';
+
+export const command = 'create <password>';
```

Comment:
As you mentioned in your TODO list we could hide this input and perhaps also confirm the password?

Reply:
Yup, although I am thinking it should be for a separate PR since the CLI is a separate tool plus I don't think it would be a trivial change.